### DISCUSSION TITLE
Download for Linux fetches old version ?

### DISCUSSION THREAD

**Discussion by pcar75** [jamulussoftware/jamulus#3412] [Technical Support & Help]
Bonjour,
I've had to reinstall Ubuntu Studio (22.04.5) and many other softwares. I reinstaleld  Jamulus from the download page of this linux variety but the version it installed is 3.8.1+dfsg1-1 (Ubuntu/amd64).  As the current version is 3.11.0 , is this normal for Linux (i.e. lagging version) ?

**Comment by rdica** [comment]
https://github.com/jamulussoftware/jamulus/releases
https://github.com/jamulussoftware/jamulus/releases/download/r3_11_0/jamulus_3.11.0_ubuntu_amd64.deb

Checked the link from https://jamulus.io and it offered 3.11.0 download for deb pkg.

**Reply by pcar75** [reply]
@rdica Thanks, I used the more obvious "https://jamulus.io/wiki/Installation-for-Linux", navigating from the main website.  Perhaps that page should be updated ? Anyone web contributor ?
I will try the newer version somewhat later ( old reflex from MS versions ending with '.0'   ;-{

**Reply by rdica** [reply]
In that page:
```
Installing the .deb files manually
Download Jamulus for your architecture: amd64, armf or arm64
```
These links all point to 3.11.0 too.
